### ISSUE TITLE
Question: For a hetero_nn model similar to the breast_csv example, can I have a csv of only the id's and labels?

### ISSUE THREAD

**Issue by PaulKMandal** [FederatedAI/FATE#4940]
Can I define a single bottom model without labeled data and then define a top model on a different federate and upload a csv with only indexes and labels to that federate? Will this work out of the box with images or will modifications need to be made? (I know by default that you guys extend ImageFolder from torchvision, I'm just asking for additional clarity.)

Thanks!

**Comment by talkingwallace** [comment]
I'm afraid I don't understand completely
By default, the hetero NN setting is:
Guest( bottom + top, feature + label)
Host( Bottom + feature）

And you want
Guest(top, feature + label)
Host(bottom, feature) ?

If so, maybe you can define a fake bottom model in guest like:

class FakeBottom(nn.Module):

    def __Init__(): xxx
    
    def forward(self, feature):
         return feature
         
Hope this can help you
